tests/storage_tests/unit_tests/test_interface.py

```diff
@@ -137,7 +137,7 @@ def test_allocate_cpu(param_dict):
 
     raw_buffer, field = allocate_cpu(shape, layout_map, dtype, alignment_bytes, aligned_index)
 
-    ## Doest this still makes sense?
+    # Does this still makes sense?
```

Comment:
What exactly? The test caught nicely the alignment bug that I fixed here. I didn't have confidence in the complicated test at first (at first I thought the test enforces too many constraints, but in the end it worked...)

Reply:
My comment was about the assert in line 142, because it is now failing due to the different way used to reshape the buffer to a tensor, but it doesn't mean the new way it's wrong, and I think that following assert with byte_bounds is more meaningful